### ISSUE TITLE
mix phoenix.gen.html doesn't properly add @primary_key to model

### ISSUE THREAD

**Issue by brado** [michalmuskala/mongodb_ecto#68]
The mongodb_ecto doc says that the following is supposed to be added to any model to account for the binary ID:   
```
@primary_key {:id, :binary_id, autogenerate: true}
```
phoenix.gen.html does not add this, with or without the --binary-id option. If this is a problem, then it is a bug in the generator. If this is no longer a problem, then it is a bug in the doc. Unfortunately, I'm having problems with the generated controller tests, and the ID is causing test failure, so I cannot determine which is the issue. Once I can get successful tests, I'll be able to better speak to which is the real problem.

**Comment by michalmuskala** [comment]
This is not an issue of the adapter. If any it is an issue of Phoenix. 

That said the proper `@primary_key` declaration should be placed in `web/web.ex` when the `phoenix.new` generator is being used with `--binary-id` option. Usage of model or controller generators assume the project was generated with that flag, what is [clearly documented](https://github.com/phoenixframework/phoenix/blob/master/lib/mix/tasks/phoenix.gen.model.ex#L62-L66): 
>  This option assumes the project was generated with the `--binary-id` option,
  that sets up models to use `binary_id` by default. If that's not the case
  you can still set all your models to use `binary_id` by default, by adding
  following to your `model` function in `web/web.ex`option or by adding
  following to the generated model before the `schema` declaration:

**Comment by brado** [comment]
I guess that's my misunderstanding -- I wasn't even aware of the `--binary-id` option on the `mix phoenix.new` command, I was using it as an option to the `mix phoenix.gen.html` command, and it didn't complain. Furthermore, I used the following command to generate my project: 
```
mix phoenix.new my_project --database mongodb
```
But looking in my `web/web.ex`, there doesn't appear to be any difference in that one file than if I generate the project using 
```
mix phoenix.new my_project --binary_id
```
These lines are already there in my project:
```
      @primary_key {:id, :binary_id, autogenerate: true}
      @foreign_key_type :binary_id
```
Using the `--database mongodb` option must do it as well. I suppose it might be useful to make mention of that in the doc, as there is no reference to using that option to generate a project. The only mention to this I found here: 

http://wsmoak.net/2015/08/31/phoenix-ecto-mongodb.html
